addons/op-badge/userscript.js

```diff
@@ -0,0 +1,17 @@
+export default async function({ addon, console }) {
+    const posts = document.querySelectorAll(".blockpost");
+
+    const op = await fetch(`https://scratchdb.lefty.one/v3/forum/topic/posts/${window.location.href.split("/")[5]}/0?o=oldest`).then(res => res.json());
+
+    for (const i of posts) {
+        if (i.querySelector(".username").innerText == op[0].username) {
+            let br = document.createElement("br");
+            let op_badge = document.createElement("span");
```

Comment:
You can probably achieve the same result (br + span) by using a div. We usually don't use <br> tags in addons but it's not enforced